gcloud-java-core/src/main/java/com/google/gcloud/Service.java

```diff
@@ -16,6 +16,6 @@
 
 package com.google.gcloud;
 
-public interface Service<O extends ServiceOptions> {
-  O options();
+public interface Service<OptionsT extends ServiceOptions<?, OptionsT>> {
```

Comment:
This type variable naming style is specific to [google](http://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s5.2.8-type-variable-names), right?
Do we use it in other open-source projects?